tests/ui/author/blocks.rs

```diff
@@ -0,0 +1,15 @@
+#![rustfmt::skip]
```

Comment:
`error[E0658]: non-builtin inner attributes are unstable`

https://travis-ci.com/rust-lang/rust-clippy/jobs/201506944#L1272

Sorry always forget about this... The `rustfmt` needs to be added to the main function.